models/stg_pinterest_ads.sql

```diff
@@ -9,6 +9,8 @@ with base as (
 
     select
         cast(campaign_date as date) as date_day,
+        cast(advertiser_id as {{ dbt_utils.type_string() }}) as accound_id,
```

Comment:
*Account_id

Reply:
Woah... cant't believe I missed that one. Thanks so much!

Reply:
```suggestion
        cast(advertiser_id as {{ dbt_utils.type_string() }}) as account_id,
```